src/main/java/frc/robot/poseestimator/IOdometryEstimator.java

```diff
@@ -13,6 +13,6 @@ public interface IOdometryEstimator {
 
 	Pose2d getOdometryPose();
 
-	void resetHeadingOffset(Rotation2d newHeading);
+	void calculateHeadingOffsetFromRobotHeading(Rotation2d newHeading);
```

Comment:
How about `setHeading`? I don't think it's that relevant for the callers of this function to know how it does what it does